apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -250,6 +253,14 @@ const MeetingForm: React.FC = () => {
         checked={keepLastFrameWhenPaused}
         onChange={toggleKeepLastFrameWhenPaused}
       />
+      <FormField
+        field={Checkbox}
+        label="Skip default meeting join device listing and selection"
+        value=""
+        checked={skipDeviceSelection}
+        onChange={toggleMeetingJoinDeviceSelection}
+        infoText="The demo will select devices on its own before meeting join"
```

Comment:
This is a bit confusing because I don't know "The demo will select devices on its own before meeting join" happens when I select this option or not.

Suggest removing this infoText.

Reply:
Ok will do.